src/bezier.jl

```diff
@@ -1,17 +1,37 @@
 const Point2d = Point2{Float64}
 
+"""
+    MoveTo(p::Point2)
+    MoveTo(x::Real, y::Real)
+
+A path command for use within a `BezierPath` which starts a new subpath at the given point.
+"""
 struct MoveTo
     p::Point2d
 end
 
 MoveTo(x, y) = MoveTo(Point2d(x, y))
 
+"""
+    LineTo(p::Point2)
```

Comment:
It looks like these methods will only take `Point2d` by the default constructor, though?  May want to add an extra `LineTo(p::VecTypes) = Point2d(p)` or something.

Reply:
all the default conversions would work

Reply:
I can change the signature so it doesn't actually look constrained in the constructor